python/lsst/ctrl/mpexec/preExecInit.py

```diff
@@ -254,7 +263,14 @@ def savePackageVersions(self, graph: QuantumGraph) -> None:
 
         # start transaction to rollback any changes on exceptions
         with self.transaction():
-            old_packages, dataset_ref = self.find_packages(graph)
+            # Packages dataset ref is stored in task init outputs, but it
+            # may be also be missing.
```

Comment:
I would have thought it was that only the packages dataset ref is stored in the `globalInitOutputs`, while the task `initOutputs` are stored elsewhere in the graph.  But maybe the QuantumGraph interface just doesn't use the nomenclature I'd expected.

Reply:
That was copy-paste from `saveConfigs`. You are right, packages ref is in `globalInitOutputs`, I'll update the comment.